package.json

```diff
@@ -19,8 +19,9 @@
   "main": "lib/index.js",
   "scripts": {
     "start": "node ./node_modules/react-native/local-cli/cli.js start",
+    "ios": "cd ./playground && react-native run-ios",
+    "android": "cd ./playground/android && ./gradlew app:assembleDebug && ./gradlew installDebug",
```

Comment:
`installDebug` task is dependent on `assembleDebug`, therefore gradle will run both when calling `installDebug`